src/main/java/org/launchcode/liftoffproject/controllers/ChildDashboardController.java

```diff
@@ -0,0 +1,100 @@
+package org.launchcode.liftoffproject.controllers;
+
+import org.launchcode.liftoffproject.data.ChildRepository;
+import org.launchcode.liftoffproject.data.ChoreRepository;
+import org.launchcode.liftoffproject.data.EarnedRewardsRepository;
+import org.launchcode.liftoffproject.models.Child;
+import org.launchcode.liftoffproject.models.Chore;
+import org.launchcode.liftoffproject.models.Reward;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import javax.servlet.http.HttpSession;
+import java.util.List;
+
+@Controller
+public class ChildDashboardController {
+
+    private final ChildRepository childRepository;
+    private final ChoreRepository choreRepository;
+    private final EarnedRewardsRepository earnedRewardsRepository;
+
+    @Autowired
+    public ChildDashboardController(ChildRepository childRepository,
+                                    ChoreRepository choreRepository,
+                                    EarnedRewardsRepository earnedRewardsRepository) {
+        this.childRepository = childRepository;
+        this.choreRepository = choreRepository;
+        this.earnedRewardsRepository = earnedRewardsRepository;
+    }
```

Comment:
In other controllers, we have
`@Autowired
private ChildRepository childRepository;` instead of creating a constructor. This might be right, but I'm wondering why it's set up differently—do you know why this way makes more sense?